init.sh

```diff
@@ -321,6 +329,20 @@ if [ $? -ne 0 ]; then
   exit 1
 fi
 
+docker pull "quay.io/decentraland/archipelago-service:${ARCHIPELAGO_DOCKER_TAG:-latest}"
```

Comment:
should we add nats here as well?

Reply:
It currently has a specific version in the `docker-compose.yaml` file, as other external services like `postgres` or `cadvisor`